src/style.css

```diff
@@ -1 +1,65 @@
-/* Styles go here */
+
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+.header {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.container {
+  max-width: 1200px;
+  min-width: 1024px;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo__link {
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+}
+
+.logo {
+  width: 100%;
+  height: 100%;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  margin: 0 50px 0 0;
```

Comment:
Remove this, use header padding instead, also reset default padding for ul - it will fix your tests